sed/loader/flash/loader.py

```diff
@@ -178,6 +177,24 @@ def available_channels(self) -> List:
         available_channels.remove("pulseId")
         return available_channels
 
+    @property
+    def schema_from_config(self) -> set:
```

Comment:
Generally, the definition of this property seems redundant. We already have available_channels and get_channels_by_format. Adding the ids to them would lead to same result, except for sector_id. 
I am thinking to just extend the latter method to include all channels and change the name to get_channels

Reply:
Updated this